src/setup.md

```diff
@@ -0,0 +1,82 @@
+## Development setup `[setup]`
+
+As a new Nimbus developer there is a lot to learn.
+
+Are you already familiar with crypto? If so, [start here](#development)
+
+Are you new to the crypto space? If so, [start here](#crypto)
+
+## Crypto
+
+At nimbus we are building a validator client for the "[Beacon Chain](https://medium.com/poloniex/what-is-ethereums-beacon-chain-fcac348210d7)".
+
+There exists a ["Proof of Work"](https://medium.com/coinmonks/simply-explained-why-is-proof-of-work-required-in-bitcoin-611b143fc3e0)" repository, the [nimbus-eth1](https://github.com/status-im/nimbus-eth1) repo.
```

Comment:
There are some factual errors in here - Ethereum is no longer using proof-of-work, we are not building just a validator client, etc, but thank you for the effort and for the feedback regarding what's needed. We'll probably be able to rework this document into something more appropriate.

Reply:
Generally, the crypto references don't belong in the _Nim_ style guide: the guide is generic for all kinds of projects - such docs should be in the nimbus-eth2/nimbus-eth1 repos respectively (https://nimbus.guide/developers.html for example).

We write the style guide for a general Nim public that wants to know how to use Nim in a production project and status itself has many non-crypto projects.